.ci/jobs/beats-mbp-2.0.yml

```
@@ -0,0 +1,63 @@
+---
+- job:
+    name: Beats/beats-mbp-2.0
+    display-name: 'Beats (2.0)'
+    description: 'Beats Main Pipeline 2.0'
+    view: Beats
+    concurrent: true
+    project-type: multibranch
+    prune-dead-branches: true
+    days-to-keep: 30
+    script-path: '.ci/Jenkinsfile'
+    triggers: []
+    wrappers: []
+    scm:
+    - github:
+        branch-discovery: 'no-pr'
+        discover-pr-forks-strategy: 'merge-current'
+        discover-pr-forks-trust: 'permission'
+        discover-pr-origin: 'merge-current'
+        head-filter-regex: '(master|7\.[x789]|8\.\d+|PR-.*|v\d+\.\d+\.\d+)'
+        discover-tags: true
+        disable-pr-notifications: true
+        notification-context: "beats-ci-2.0"
+        repo: 'beats'
+        repo-owner: 'elastic'
+        credentials-id: github-app-beats-ci
+        ssh-checkout:
+          credentials: f6c7695a-671e-4f4f-a331-acdce44ff9ba
+        property-strategies:
+          all-branches:
+          - suppress-scm-triggering: true
+        build-strategies:
+        - skip-initial-build: true
```

Aha, this will run only for those items with the `.ci/Jenkinsfile` so we are safe for now

```suggestion
        build-strategies:
```